air.toml

```diff
@@ -0,0 +1,8 @@
+[format]
```

Comment:
should probably add this to `.gitignore`? Specific to your setup?

Reply:
It's pretty early, but these are some formatting settings that match our styler settings. (If you haven't seen, the [air formatter](https://www.tidyverse.org/blog/2025/02/air/) is pretty new, but likely to be a common future formatter in R. Logan and I are both using it.)